iOS/MMIWG2S/Utilities/Extensions.swift

```diff
@@ -99,3 +121,7 @@ extension UIView {
     }
     
 }
+
+public func * (size: CGSize, scalar: CGFloat) -> CGSize {
+    return CGSize(width: size.width * scalar, height: size.height * scalar)
+}
```

Comment:
Well that's cool and also odd you had to create your own multiplication function.

Reply:
yeah swift only provides the basics by default